components/sysutils/net-snmp/Makefile

```diff
@@ -73,15 +72,15 @@ CONFIGURE_OPTIONS  +=	--with-persistent-directory=$(VARDIR)/net-snmp
 
 CONFIGURE_OPTIONS  +=	--with-mibdirs=$(ETCDIR)/net-snmp/snmp/mibs
 CONFIGURE_OPTIONS  +=	--datadir=$(ETCDIR)/net-snmp
+CONFIGURE_OPTIONS  +=	--enable-agentx-dom-sock-only
 CONFIGURE_OPTIONS  +=	--enable-ucd-snmp-compatibility
 CONFIGURE_OPTIONS  +=	--enable-ipv6
 CONFIGURE_OPTIONS  +=	--enable-mfd-rewrites
 CONFIGURE_OPTIONS  +=	--with-openssl=/lib/openssl
-CONFIGURE_OPTIONS  += --sysconfdir=$(ETCDIR)/net-snmp
```

Comment:
Why configuration files are moved out of /etc?

Reply:
This is the same location as the default. 
For teh sake of clarification we can add this again:
CONFIGURE_OPTIONS  += --sysconfdir=$(ETCDIR)/net-snmp

Reply:
Can you readd ti?